tsconfig.json

```diff
@@ -31,7 +31,7 @@
   "typedocOptions": {
     "out": "dist/docs/typescript",
     "exclude": "**/*.spec.ts",
-    "theme":  "node_modules/ig-typedoc-theme/dist",
+    "theme":  "node_modules/igniteui-typedoc-theme/dist",
```

Comment:
Adding `"excludeExternals": true,` here seems to remove the HTMLElement derived properties, it also fixes the messy property/method categories.